README.md

```diff
@@ -109,15 +109,63 @@ Endorsement requests will be auto-endorsed if the `ENDORSER_AUTO_ENDORSE_REQUEST
 
 ### Granular Configuration of Auto Endorsement
 
-Auto endorsement of transactions can be configured via the `/allow/*` endpoints
+Auto endorsement of transactions is configured via the `/allow/{publish-data,schema,credential-definition}` endpoints
 
 Each endpoint supports a `GET`, `POST` and `DELETE` for listing the
-allowed automatically endorsable transactions, adding new transaction
-to be automatically endorsed, and delete transactions.
+allowed automatically endorsable transactions, adding new transactions to be automatically endorsed, and deleting transactions.
 
-Any requests using the `POST` method supports using "*" to indicate a
+Any requests using the `POST` method support using "*" to indicate a
 wild card.
 
+#### Configuration with CSV files
+
+Auto endorsement of transactions can be configured via the `/allow/config` endpoints
+
+This endpoint supports `PUT` and `POST` which will allow you to bulk modify a list of allowed automatically endorsable transactions.
+
+- POST: This method will replace the current configuration with the data from the uploaded CSV file.
+
+- PUT: In contrast, the PUT method appends the data from the CSV file to the existing configuration, preserving the current state.
+
+Each of these endpoints supports uploading a CSV file for `publish-data`, schema, and `credential-definition`.
+
+The fields of these CSVs follow the format used in the `POST /allow/{publish-data,schema,credential-definition}` endpoints
+
+For example, the description for the `POST /allow/`schema` endpoint and the CSV equivalent is
+
+| Name          | Description | default           |
+|---------------|-------------|-------------------|
+| `author_did`  | string      | Default value : * |
+| `schema_name` | string      | Default value : * |
+| `version`     | string      | Default value : * |
```

Comment:
Examples of all the expected structures for the csv files (do we need a header? what are the columns?) would be good to help users know what to upload, since the API takes a file and cannot document it.